.github/workflows/push-master.yml

```diff
@@ -7,18 +7,88 @@ on:
 jobs:
```

Comment:
I don't think we want to modify the github actions workflows here. Ours differs from Parity's